src/api-endpoints.ts

```diff
@@ -15,4 +16,8 @@ export default class ApiEndpoints {
   ufcEndpoint(): URL {
     return this.endpoint(UFC_ENDPOINT);
   }
+
+  banditParametersEndpoint(): URL {
+    return this.endpoint(BANDIT_ENDPOINT);
```

Comment:
Kudos to @felipecsl for making this class. It made it super easy to add a new endpoint without worrying about host and stuff.

Reply:
🫡